instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py

```diff
@@ -197,6 +206,14 @@ def _wrap_perform_request(
 ):
     # pylint: disable=R0912,R0914
     def wrapper(wrapped, _, args, kwargs):
+        # if wrapped elasticsearch has native OTel instrumentation just call the wrapped function
+        otel_span = kwargs.get("otel_span")
+        if otel_span and otel_span.otel_span:
+            warnings.warn(
+                "Using native elasticsearch OTel instrumentation", Warning
```

Comment:
This isn't enough, we should:

 * say that the instrumentation for this package is disabled in favor of the Elasticsearch package
 * ideally link to a page that contains the information 
 * ideally figure out the correct stacklevel so that the warning appears in user code

Here's a good example: https://github.com/urllib3/urllib3/blob/b8589ec9f8c4da91511e601b632ac06af7e7c10e/src/urllib3/poolmanager.py#L422-L430

Reply:
Added 1) and 2). For 3) I prefer not poking with stacktraces since this is a _service_ issue not local to some callers.